settings.gradle

```diff
@@ -32,3 +32,6 @@ project(":frameworks/prototype").name = "prototype"
 
 include 'frameworks/proxylite'
 project(":frameworks/proxylite").name = "proxylite"
+
+include 'examples/gpuresource'
+project(":examples/gpuresource").name = "gpuresource"
```

Comment:
It is unnecessary to add a whole new project.  Just specify a different `examples/gpu.yml` file for the `helloworld` project and use it in an integration test.  Similar to the [CNI integration test example](https://github.com/mesosphere/dcos-commons/blob/master/frameworks/helloworld/tests/test_networking.py#L18).